src/ucp/wireup/select.c

```diff
@@ -510,14 +537,19 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
                 continue;
             }
 
-            score        = criteria->calc_score(wiface, md_attr,
-                                                &ae->iface_attr);
+            score_factor = local_factor *
+                           ucp_wireup_score_factor(wiface, remote_dev_counts,
```

Comment:
can we make ucp_wireup_score_factor part of calc_score callback so the common function ucp_wireup_select_transport would not need to change?